### ISSUE TITLE
Last Login field does not sort properly

### ISSUE THREAD

**Issue by rak-phillip** [rancher/dashboard#10821]
Sorting for Last Login field in the Users table will sort by alphabetical order instead of the date. 

![image](https://github.com/rancher/dashboard/assets/835961/07d79f80-080e-4436-8ea9-f2c06f21b316)

## Steps to Reproduce

1. Create 2 local users
2. Login as both the users
3. From users and Authentication >> Users >> sort by last login

## Expected Behavior

Last Login is properly sorted by Ascending/Descending based on the date. 

## Actual Behavior

Last Login sorts by Ascending/Descending based on the alphabetical order.

**Comment by IsaSih** [comment]
Tests fail on v2.8.s4-8057cd39a426289bf7c6cbe5f09f9062d9183e2b-head. The users table is sorted by username alphabetical order, not by the latest login, as demonstrated in the scree

![Image](https://github.com/rancher/dashboard/assets/70187881/47835799-edcf-4ad0-b4ab-adc8f04a3fe4)

**Comment by rak-phillip** [comment]
@IsaSih This issue is about ensuring that when we click on the "Last Login" header that it sorts users by the actual time that they were last logged in. In the screenshot from the description above, the entry for 51 minutes looks larger than the entry for 5 hours when sorting the Last Login column. This should no longer be the case.

### Descending
![image](https://github.com/rancher/dashboard/assets/835961/b40eaa7c-9660-4a91-b8a4-6c10f35818ec)

### Ascending
![image](https://github.com/rancher/dashboard/assets/835961/86123bd3-86d3-47af-8767-c561262664b7)

**Comment by IsaSih** [comment]
Ok, understood. Tests pass on v2.8.s4-8057cd39a426289bf7c6cbe5f09f9062d9183e2b-head

**Comment by dnoland1** [comment]
This still does not work in v2.9.1. What I see in 2.9.1:
![image](https://github.com/user-attachments/assets/c1cbdb70-b95a-4f5f-b84c-539f23741b03)

**Comment by jbilliau-rcd** [comment]
Ha, I just made a case about this very thing @dnoland1 . It works in our sandbox env, but not nonprod....even though they both the same version. The one difference I can see is, for sandbox, we updated ALL users to have a value for the `user-last-login-default` setting.

```
apiVersion: management.cattle.io/v3
customized: true
default: null
kind: Setting
metadata:
  generation: 13
  name: user-last-login-default
source: null
value: "2024-08-16T20:00:00-04:00"
```

And sorting works....for nonprod only users who have logged in have a "last login" value, it doesn't. So my theory is sorting only works if everyone has a value....the sort code doesn't like users with null values.
